azkaban-exec-server/src/main/java/azkaban/container/FlowContainer.java

```diff
@@ -190,9 +200,14 @@ public FlowContainer(final Props props,
       @Named(EXEC_CONTAINER_CONTEXT) final Context context) throws ExecutorManagerException {
 
     // Create Azkaban Props Map
-    this.azKabanProps = props;
+    this.azkabanProps = props;
+    // Get the execution ID from the environment
+    this.execId = getExecutionId();
+    // Set up RootLogger
+    createLogger();
+
     // Setup global props if applicable
-    final String globalPropsPath = this.azKabanProps.getString("executor.global.properties", null);
+    final String globalPropsPath = this.azkabanProps.getString("executor.global.properties", null);
```

Comment:
how about making "executor.global.properties" constant?

Reply:
done. Thanks for bringing this up!